.github/workflows/switch.yml

```diff
@@ -0,0 +1,112 @@
+name: Switch workflow branches
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "Version to check"
+        required: false
+
+env:
+  NEXT_TAG: ${{ github.event.inputs.version }}
+
+jobs:
+  branch-switch:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/checkout@v3
+        with:
+          path: '/tmp/gitpod/'
+          repository: 'gitpod-io/gitpod'
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+      - name: Install dependencies
+        run: npm i
+      - name: Run the branch switch-a-roo
+        uses: actions/github-script@v6
+        with:
+          script: |
+            import semver from 'semver';
```

Comment:
Same here, I have not tested it before but I think you need to use `require` instead of import

Reply:
Looks like you're correct, thank you! https://github.com/actions/github-script#use-esm-import